spark_connect_labextension/app.py

```diff
@@ -0,0 +1,56 @@
+from jupyter_server.extension.application import ExtensionApp
+from spark_connect_labextension.handlers.cluster.start_cluster import StartClusterRouteHandler
```

Comment:
Could be done, but you'd still need one line per import since those handlers are all in different modules, so I'm not sure if there is any real gain if those imports are not used elsewhere too?

What could be done is shorten those lines with explicit relative imports, e.g.:

```python
from .handlers.cluster.stop_cluster import StopClusterRouteHandler
```

Reply:
Okay I'll refactor to use relative imports